inc/fluent_bit/class-msgpackformatter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Altis Cloud Fluent Bit MsgPack formatter.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Fluent_Bit;
+
+/*
+ * TODO Copies some formatter from Monolog, need to give credit
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Monolog\Formatter\FluentdFormatter;
+use RuntimeException;
+
+/**
+ * Class MsgPackFormatter
+ *
+ * Serializes a log message to Fluent Bit socket protocol
+ *
+ * @author Nathaniel Schweinberg <nathaniel@humanmade.com>
```

Comment:
Individual credit not needed, work is copyright Human Made and handled at the package level.

```suggestion
```